dicom-archive/get_dicom_info.pl

```diff
@@ -291,6 +478,19 @@ sub InfoOption {
     1;
 }
 
+=pod
+
+=head3 TwoArgInfoOption($option, $rest)
+
+Greps the group and element information from the GetOpt table options specified
+and check that the two arguments required by the option has been set.
```

Comment:
...and check**s** that the two arguments required by the option **have** been set

Reply:
done